vllm/model_executor/layers/sampler.py

```diff
@@ -38,12 +38,15 @@ def forward(
         embedding: torch.Tensor,
         hidden_states: torch.Tensor,
         input_metadata: InputMetadata,
+        embedding_bias: Optional[torch.Tensor] = None,
```

Comment:
Should this change be in a separate PR? Seems not related to GPT-J.

Reply:
BTW I think this `embedding_bias` is very nice! We should add this to support `logit_bias` in OpenAI API: https://platform.openai.com/docs/api-reference/chat/create

Reply:
This is relevant because GPT-J has a bias term in its `lm_head`. But yes, it may also be used for `logit_bias` in the future.

Reply:
I see. Thanks!

Reply:
Or, maybe we should have `logit_bias` as a separate argument, because `embedding_bias` is applied to all sequences while `logit_bias` can be different by sequences in the batch.

Reply:
Yes. I think keeping the argument name here as `embedding_bias` is good. `logit_bias` should be in `sampling_params`.